pages/index.vue

```diff
@@ -27,8 +33,6 @@
 
 <script setup>
 import viewer_schemas from "@geode/opengeodeweb-viewer/schemas.json";
-import Status from "@ogw_f/utils/status.js";
```

Comment:
I think you need that

Reply:
Not auto imported, because not directly in vease but ogw_f, maybe we should auto import them